app/models/core_induction_programme_exporter.rb

```diff
@@ -8,22 +8,46 @@ def run
       exclude: %i[created_at updated_at],
       import: true,
     )
+
+    course_years = CourseYear.order(:created_at)
+    SeedDump.dump(
+      course_years,
+      file: "db/seeds/cip_seed_dump.rb",
+      exclude: %i[created_at updated_at],
+      import: true,
+      append: true,
+    )
+
+    course_modules = []
+    course_years.each do |course_year|
+      course_modules += course_year.course_modules.order(:created_at)
+    end
     SeedDump.dump(
-      CourseYear.all,
+      course_modules,
       file: "db/seeds/cip_seed_dump.rb",
       exclude: %i[created_at updated_at],
       import: true,
       append: true,
     )
+
+    course_lessons = []
```

Comment:
We could use reduce /  inject. Probably more rubyscue

Reply:
Ruby has two methods for reducing? :O 

I went with `map` + `flatten`. I think it's easier to understand than reduce.

Reply:
Yes, the usual ruby thing about coder happiness. Apparently, inject doesn't feel natural to advocates of reduce. I think one is just an alias to the other though.